### DISCUSSION TITLE
Fail under docker alpine image.

### DISCUSSION THREAD

**Discussion by MedinskyAlexander** [devopvoid/webrtc-java#153] [Q&A]
Hi, I'm working on spring based application that connects to livekit server as a client and receives chat messages, it means I don't need any audio/video support. It works as expected locally (Ubuntu 24.04) but crashes under docker (tried: alpine, amazoncorretto:18-alpine, amd64/ubuntu) container with error:
`java.lang.UnsatisfiedLinkError: /tmp/libwebrtc-java13733588589512987869.so: Error loading shared library libpulse.so.0: No such file or directory (needed by /tmp/libwebrtc-java13733588589512987869.so)`

Fail happens on new PeerConnectionFactory(). _pulseaudio_ installation did not help. 
Can you assist please to disable audio properly or point to suitable docker image for my app?

**Comment by devopvoid** [comment]
You can try [disabling the OS audio layer](https://jrtc.dev/#/guide/audio_devices?id=disabling-audio).

**Reply by Kas-tle** [reply]
Hello, I have tried using `kDummyAudio` as you describe in the docs. This still results in a `java.lang.UnsatisfiedLinkError`. This makes sense, as there is a static block in `AudioDeviceModuleBase` that loads all natives:
https://github.com/devopvoid/webrtc-java/blob/main/webrtc/src/main/java/dev/onvoid/webrtc/media/audio/AudioDeviceModuleBase.java#L44-L51

And pulse is required on linux here in your CmakeList: https://github.com/devopvoid/webrtc-java/blob/main/webrtc-jni/src/main/cpp/CMakeLists.txt#L113
